version/version.go

```diff
@@ -6,7 +6,7 @@ const (
 	// VersionMajor is for an API incompatible changes
 	VersionMajor = 5
 	// VersionMinor is for functionality in a backwards-compatible manner
-	VersionMinor = 33
```

Comment:
I don't think we want this.  Should main be at 5.33.*-dev?

Reply:
The net effect of this PR should be to keep main at 5.33.0-dev , as in https://github.com/mtrmac/image/blob/into-main/version/version.go (and the PR’s file change view is showing no changes).

The PR contents are all contents of the `release-5.32` branch, + a merge commit into `main` (it needs to be that way, the goal is to make the commit tagged 5.32.2 an ancestor of the `main` branch).

So, we do include the 5.32 version bumps as individual commits, but the merge commit undoes the `version.go` changes.